src/main/java/teammates/common/datatransfer/ResultFetchType.java

```diff
@@ -0,0 +1,34 @@
+package teammates.common.datatransfer;
+
+/**
+ * The result fetching type to indicate whether the fetch is for giver only, receiver only or both.
```

Comment:
Nit but can I trouble you to add in "feedback" in front of giver and receiver here just for more clarity when viewing this file in isolation

Reply:
Thanks for your insight! I am wondering if this enum can be used for general purpose and hence left out the feedback part here. In the meantime I will just add it in!